pkg/registry/servicecatalog/instance/storage.go

```diff
@@ -147,6 +149,38 @@ func NewStorage(opts server.Options) (rest.Storage, rest.Storage, rest.Storage)
 		DeleteStrategy:          instanceRESTStrategies,
 		EnableGarbageCollection: true,
 
+		TableConvertor: tableconvertor.NewTableConvertor(
+			[]metav1beta1.TableColumnDefinition{
+				{Name: "Name", Type: "string", Format: "name"},
+				{Name: "Class", Type: "string"},
```

Comment:
We should think about how we want to differentiate ns/cluster scoped classes/plans and requested vs. matched service/plan here

Reply:
I suggest not differentiating for now and just printing both cluster and ns-scoped classes/plans in the same column. I definitely don't want to see two different columns for this, as that would make the output too wide. 

But we could add a prefix to the values in the column (e.g. `csc/class-name` for clusterserviceclasses and `sc/class-name`for serviceclasses)

Reply:
FYI, a couple of us talked about what this should look like in svcat at the last F2F, here are notes:

https://github.com/kubernetes-incubator/service-catalog/pull/1826#issuecomment-382148545

Reply:
After thinking about it more, I would much rather prefer that we display the namespace (and leave it empty for cluster level resources) than use the prefixes.

Reply:
Yeah, I don't like the prefixes idea either. But I also don't like displaying the namespace, since a ServiceInstance never references a ServiceClass/Plan in a different namespace. It's either the the namespace of the ServiceInstance or no namespace. 

Actually, I've just remembered that in core k8s, a `RoleBinding` can reference either a `ClusterRole` or a namespaced `Role`. This is how kubectl prints the reference:
```
$ kubectl get rolebinding -o wide
NAME       AGE       ROLE                               USERS     ...
test       1m        Role/my-namespaced-role            foo                 
test2      2s        ClusterRole/my-cluster-role        foo           
```

So, to be consistent, we need to prefix with `ServiceClass/` and `ClusterServiceClass/`.